README.md

```diff
@@ -87,6 +87,7 @@ Commands:
   aptible metric_drain:deprovision HANDLE --environment ENVIRONMENT                                                                                                                                                                                     # Deprovisions a Metric Drain
   aptible metric_drain:list                                                                                                                                                                                                                             # List all Metric Drains
   aptible operation:cancel OPERATION_ID                                                                                                                                                                                                                 # Cancel a running operation
+  aptible operation:connect OPERATION_ID                                                                                                                                                                                                                # Follow logs of a running operation
```

Comment:
nitpicky - if it's log related, i might call it `:follow` or `:tail`, not sure. Or I would change the doc to follow what it says `Connect and follow` or something to that effect

Reply:
Updated to `:follow`